components/ItemsPage/ItemsPage.tsx

```diff
@@ -28,7 +28,7 @@ const ItemsPage = (props: Props) => {
    const { menuItems, toppings, sauces, menuItemOptions } = useContext(menuContext)
 
    const { settingsModel } = useContext(settingsContext)
-   const maxAmount = settingsModel?.maxAmountOfAddons ? settingsModel?.maxAmountOfAddons : 0
+   const maxAmount = settingsModel?.maxAmountOfAddons ? settingsModel?.maxAmountOfAddons : undefined
```

Comment:
Why are you setting `maxAmount` to `undefined` instead of setting it to zero? I'm asking because I might be missing some of how the new logic is working. Setting to undefined something that was previously always a number may open a window to the introduction of bugs. Might not be this case though. Why did you have to set it to `undefined`